### ISSUE TITLE
Iron router splash screen instead of app routes on mobile and Windows Chrome

### ISSUE THREAD

**Issue by jjman505** [iron-meteor/iron-router#1290]
Instead of our normal routes, we're getting the splash screen on Chrome for Windows and all mobile browsers. Live at https://dunmoapp.com. This error also occurs on IE11 technical preview.

What could be causing this?

**Comment by cmather** [comment]
Any errors in the server log? Bunch of people seem to be seeing this. I suspect there's some kind of error that results in no routes being created. Not sure why it's only an issue in production. I might have to dig into this one. Anyone else figure out a deterministic cause?

**Comment by jjman505** [comment]
I checked my logs, and I don't see any errors. Totally clueless on this one. :\

**Comment by yanndebelgique** [comment]
Yup same problem with me too. Only happens in production for certain browsers :-(
Same issue here as well in the mup project related to this : https://github.com/arunoda/meteor-up/issues/333

**Comment by ofthewood** [comment]
Same problem. Is the a workaround ? 
thks.

**Comment by yanndebelgique** [comment]
Ok so workaround is to put code in meteor.startup. to know which code you need to wrap you need to rip out chunks and run it on the ios simulator in production mode to see when issue disappears. I zeroed in on the offending code, wrapped it in meteor.startup and solved the issue. hope this helps
